clang-tools-extra/clang-doc/tool/ClangDocMain.cpp

```diff
@@ -133,17 +133,17 @@ std::string getFormatString() {
 // GetMainExecutable (since some platforms don't support taking the
 // address of main, and some platforms can't implement GetMainExecutable
 // without being given the address of a function in the main executable).
-std::string GetExecutablePath(const char *Argv0, void *MainAddr) {
+std::string getExecutablePath(const char *Argv0, void *MainAddr) {
   return llvm::sys::fs::getMainExecutable(Argv0, MainAddr);
 }
 
-llvm::Error GetAssetFiles(clang::doc::ClangDocContext &CDCtx) {
+llvm::Error getAssetFiles(clang::doc::ClangDocContext &CDCtx) {
   std::error_code Code;
   for (auto DirIt = llvm::sys::fs::directory_iterator(UserAssetPath, Code),
             DirEnd = llvm::sys::fs::directory_iterator();
-       !Code && DirIt != DirEnd; DirIt.increment(Code)) {
+       DirIt != DirEnd; DirIt.increment(Code)) {
     llvm::SmallString<128> FilePath = llvm::SmallString<128>(DirIt->path());
-    if (!Code) {
+    if (Code) {
       return llvm::createFileError(FilePath, Code);
     }
```

Comment:
nit: we can drop the braces for a single statement conditional, since it's part of the style guide.